Riot/Modules/Room/Views/Title/Preview/PreviewRoomTitleView.m

```diff
@@ -183,29 +183,10 @@ - (void)refreshDisplay
         {
             MXStrongifyAndReturnIfNil(self);
 
-            NSArray *members = roomMembers.members;
-            NSUInteger activeCount = 0;
-            NSUInteger memberCount = self.mxRoom.summary.membersCount.joined;
-            NSString *inviter = nil;
-
-            for (MXRoomMember *mxMember in members)
-            {
-                if (mxMember.membership == MXMembershipJoin)
-                {
-                    // Get the user that corresponds to this member
-                    MXUser *user = [self.mxRoom.mxSession userWithUserId:mxMember.userId];
-                    // existing user ?
-                    if (user && user.presence == MXPresenceOnline)
-                    {
-                        activeCount ++;
-                    }
-
-                    // Presently only one member is available from invited room data
-                    // This is the inviter
-                    inviter = mxMember.displayname.length ? mxMember.displayname : mxMember.userId;
-                }
-            }
-
+            MXRoomMember *myMember = [roomMembers memberWithUserId:self.mxRoom.mxSession.myUserId];
+            NSString *inviterUserId = myMember.originalEvent.sender;
+            NSString *inviter = [roomMembers memberName:inviterUserId] ?: inviterUserId;
```

Comment:
We should try to resolve the display name a bit more.
if `[roomMembers memberName:inviterUserId]` is `nil`, we should try to use  `[mxSession userWithUserId: inviterUserId]``

Note: I said without checking last spec information. I tend to not trust the HS :)